docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.8'
+
+services:
+  auth:
+    image: node:latest
+    working_dir: /app
+    volumes:
+      - ../WeFriiendsAuth:/app
+    command: npm run start:dev
+    ports:
+      - "3001:8080" # Map the port to access the service if needed
+    environment:
+      - GOOGLE_AUTH_CLIENT_ID=replace-me
+      - GMAIL_USER=replace-me
+      - GMAIL_PASSWORD=replace-me
+      - GMAIL_CLIENT_ID=replace-me
+      - GMAIL_CLIENT_SECRET=replace-me
+      - GMAIL_REFRESH_TOKEN=replace-me
+      - STORAGE_CONNECTION_STRING=mongodb://wefriiends:women2022@storage/authorization?authSource=admin&retryWrites=true&w=majority
+    depends_on:
+      - storage
+
+  frontend:
+    image: node:latest
+    working_dir: /app
+    volumes:
+      - ./:/app
+    command: npm start
+    ports:
+      - "3000:3000" # Map the port to access the frontend
+    environment:
+      - REACT_APP_AUTH_SERVICE_HOST=http://localhost:3001
+
+  storage:
+    image: mongo:latest
+    ports:
+      - "27017:27017" # Default MongoDB port
+    volumes:
+      - mongodb_data:/data/db
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=wefriiends
+      - MONGO_INITDB_ROOT_PASSWORD=women2022
```

Comment:
I don't think it's a good idea to write them openly

Reply:
@Svitlana-G credentials are not from cloud DB, it's locally created